Resources/Prototypes/_CP14/Entities/Structures/crystal.yml

```diff
@@ -11,12 +11,23 @@
       - state: big
         map: ["random"]
         shader: unshaded
-    noRot: true
+    noRot: true   
+  - type: InteractionPopup
+    interactSuccessString: popup-cp14crystal-ding
+    messagePerceivedByOthers: popup-cp14crystal-ding
+    interactSuccessSound:
+      collection: crystaldings
+      params:
+        variation: 0.03
   - type: Physics
```

Comment:
Some components are redundant after changing the parent

Reply:
!

Reply:
You mean Crystaldings?

Reply:
![image](https://github.com/crystallpunk-14/crystall-punk-14/assets/92464780/21ce4264-b234-49a3-990b-f9ece8e71fce)
We already changed that

Reply:
nope, i about static body